zipline/resources/security_lists/leveraged_etf_list/20020103/20160907/add

```diff
@@ -0,0 +1,35 @@
+EUFL
+REML
+HYDD
+AMJL
+FIHD
+FIEE
+MLPZ
+MLPQ
+GASX
+SICK
+LBDC
+MRRL
+PILS
+HAKK
+HAKD
+PILL
+NAIL
+CLAW
+DPST
+WDRW
+UBIO
+ZBIO
+SOP
+UOP
+LABU
+LABD
+GUSH
+DRIP
+LRET
+CHAU
+HOML
+GDXX
+GDJJ
+GDXS
+SMHD
```

Comment:
add a newline then this looks functionally looks fine, no idea if these tickers are actually leveraged etfs.